build.gradle

```diff
@@ -1,21 +1,26 @@
+import com.github.jk1.license.filter.LicenseBundleNormalizer
+import com.github.jk1.license.render.JsonReportRenderer
+
 plugins {
     id 'java-platform'
     id 'maven-publish'
-    id "io.github.gradle-nexus.publish-plugin" version "1.3.0"
+    id "io.github.gradle-nexus.publish-plugin" version "2.0.0"
     id 'signing'
-    id "org.owasp.dependencycheck" version "9.0.9"
-    id "com.gorylenko.gradle-git-properties" version "2.4.1"
+    id 'org.owasp.dependencycheck' version "10.0.4"
+    id "com.gorylenko.gradle-git-properties" version "2.4.2"
+    id 'com.github.jk1.dependency-license-report' version "2.9"
+    id 'de.undercouch.download' version "5.6.0"
 }
 
 ext {
-    slf4jVersion = '2.0.7'
-    log4jVersion = '2.23.0'
-    grpcVersion = '1.62.2'
-    protobufVersion = '3.25.3'
-    jacksonVersion = '2.16.1'
+    slf4jVersion = '2.0.16'
+    log4jVersion = '2.23.1'
+    grpcVersion = '1.66.0'
```

Comment:
note: we will also need to update grpc versions in th2 gradle plugin

Reply:
Yes, I would like to approve dependabot's suggestion